samples/solidity/test/index.ts

```diff
@@ -0,0 +1,19 @@
+import { expect } from "chai";
+import { ethers } from "hardhat";
+
+describe("Greeter", function () {
```

Comment:
Can probably remove the test folder if we're not adding real tests at this time.